### PR TITLE
split react preview components out into separate files

### PR DESCRIPTION
**Pull request by emma-floored**
@hwigmore 

I think this could be considered part of the ticket you made for eventually including react previews on regular doc pages. I split out each chunk of the big `preview.jsx` file into its own file, which will also hopefully make updating that page less painful in the future.

### CODE REVIEW

File: docs/_javascript/AccordionPreview.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const React = require('react');
+
+const BlocksReact = require('../../react-index.js');
+const AccordionSection = BlocksReact.Accordion;
```
Comment (hwigmore) [inline docs/_javascript/AccordionPreview.jsx]: this is _really_ not a big deal at all but in all these files you might as well just 
`const AccordianSection = require('../../react-index').Accordian` so you aren't importing way more than is necessary

Comment (emma-floored) [reply docs/_javascript/AccordionPreview.jsx]: oh yeah, I was just lazy and copied from when it was all in one file, but your way is better :P

**Comment by emma-floored** [comment]
@hwigmore made both those changes!

**Comment by hwigmore** [comment]
👍

**Comment by hwigmore** [comment]
thank you for doing this, by the way!!

**Pull request merged**